tixl/eth-oracle-tx-service
# All 4 API methods tested

All 4 API methods should work correctly now.

Test program:
1. create a config file "settings.json" in project root, containing:
{
  "address":"0x0dF22FdeD1D944365B6561B7a620a48E3f4D3CCe", // replace with pool address
  "collect":"0xF86B8D1BEE8866e9E3cAAE0004A333C288Dd7c5f",
  "server":"http://207.154.253.13",
  "signer":"localhost",
  "mnemonic": "word1 word2 .... word12" // replace with your own wallet
}
2. run: npx ts-node -T
3. in the ts-node shell, run: .load test_simple.ts
4. invoke one of the helper methods in test_simple.ts (testTx will send money from pool address to another address, testOracle will get tx info, testVerify will check signature)
